packages/engine/src/avatar/AvatarControllerSystem.ts

```diff
@@ -68,7 +69,34 @@ export const AvatarControllerSystem = async (): Promise<System> => {
       const raycastComponent = getComponent(entity, RaycastComponent)
 
       // iterate on all collisions since the last update
-      controller.controller.controllerCollisionEvents?.forEach((event: ControllerHitEvent) => {})
+      controller.controller.controllerCollisionEvents?.forEach((event: ControllerHitEvent) => {
+        let entity = event.body.userData.entity
+        let triggerComponent = getComponent(entity, TriggerVolumeComponent)
+        if (triggerComponent) {
+          if (!triggerComponent.active) {
+            triggerComponent.active = true
+            triggerComponent.onTriggerEnter()
```

Comment:
These triggers should be component add and remove queries. Integrating with the existing InteractiveSystem would be great.

Reply:
ok will do, can you point me to an example where it's being followed already. thanks